src/lib.rs

```diff
@@ -175,6 +175,266 @@ pub fn builder() -> Builder {
 }
 
 impl Builder {
+    /// ```ignore
+    /// use bindgen::builder;
+    ///
+    /// // Configure and generate bindings.
+    /// let mut builder_1 = builder();
+    ///
+    /// // generate the command line flags use for creating `builder_1`
+    /// Builder::command_line_flags(&mut builder_1)
+    /// ```
+    pub fn command_line_flags(&self) -> Vec<String> {
+        let mut output_vector: Vec<String> = Vec::new();
+
+        if let Some(ref header) = self.options.input_header {
+            //positional argument 'header'
+            output_vector.push(header.clone().into());
+        }
+
+
+        self.options
+            .bitfield_enums
+            .get_items()
+            .iter()
+            .map(|item| {
+                     output_vector.push("--bitfield-enum".into());
+                     output_vector.push(item.clone());
+                 })
+            .collect::<Vec<()>>();
```

Comment:
Rather than collecting as a `Vec` to perform `map`'s side effects, the `Iterator::count` method is more idiomatic for forcing side effects. Even better in this case would be a plain `for` loop:

```rust
for item in self.options.bitfield_enums.get_items() {
    output_vector.push("--bitfield-enum".into());
    output_vector.push(item.clone());
}
```